src/boot.lisp

```diff
@@ -515,6 +510,17 @@
     (t
      nil)))
 
+(defparameter *features* '(:jscl :common-lisp))
+
+;;; Early error definition.
+(defun error (fmt &rest args)
+  (let ((msg fmt))
+    (if (fboundp 'format)
+        (progn
+          (setq msg (apply #'format nil fmt args))
+          (/debug msg))
+        (%console msg args)))
+  (%throw "ERROR!"))
```

Comment:
These functions `/debug` & `%console` can be used like this.
The `format` is not yet bounded, and the `error` is logged to the `console`.
I would not make these changes to the `/log` or `/debug`. 
Did as done.

Reply:
I don't think I understand.  The formatting is hapenning here in `error`.  `%console` itself isn't doing formatting,  the browser is in that sense, right?

So I don't see any fundamental difference between `%console` and `/log` or `/debug`.

Reply:
(error fmt a1 ... an)
 1. (/debug fmt) 
    (dolist (it args) (/debug it)) 

 2. (/log fmt)
    (dolist (it args) (/log it))

 3. (%console fmt args)

The differences are not fundamental, but

write multiple debug forms and delete those lines after debugging
                                                `vs`
write one form for multiple objects, delete one form after debugging

Something like this

Reply:
I agree, you're right  - `more general and basically compatible version`.
I will rename the function to `/log` and do commit.